plugins/csv/komand_csv/actions/json_to_csv_bytes/action.py

```diff
@@ -0,0 +1,20 @@
+import insightconnect_plugin_runtime
+from .schema import JsonToCsvBytesInput, JsonToCsvBytesOutput, Input, Output, Component
+
+# Custom imports below
+import base64
+from komand_csv.util.utils import json_to_csv
+
+
+class JsonToCsvBytes(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="json_to_csv_bytes",
+            description=Component.DESCRIPTION,
+            input=JsonToCsvBytesInput(),
+            output=JsonToCsvBytesOutput(),
+        )
+
+    def run(self, params={}):
+        encoded = base64.encodebytes(json_to_csv(params.get(Input.JSON)).encode())
```

Comment:
Split out this code and put a try: except: around the encodebytes action. We believe this can result in a Unicode encoding exception. (I think).

Reply:
Also, can you add a comment in the code about how this works. It's too many encodes and decodes in one line for my hamster brain to process.

Reply:
Or split this code into separate lines so it's clearer what's going on.

Reply:
Thanks for a review! I splited this code to make it more clearer. I didn't add try: except:, because the `encodebytes` method supports Unicode. I added a unit test for such a case.